src/domains/sign/io_sign_user_func.tf

```diff
@@ -0,0 +1,93 @@
+locals {
+  io_sign_user_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME                        = "node"
+      WEBSITE_NODE_DEFAULT_VERSION                    = "16.13.0"
```

Comment:
from the last issue ticket with Microsoft support they say that we need to remove WEBSITE_NODE_DEFAULT_VERSION and use only linux_fx_version for functions app on linux plan